### ISSUE TITLE
[Bug Report] Fullscreen dialog shows html/body's scrollbar after inner dialog is closed

### ISSUE THREAD

**Issue by 185driver** [vuetifyjs/vuetify#5356]
### Versions and Environment
Vuetify: 1.3.0
Vue: 2.5.17
Browsers: Chrome
OS: Windows 10

### Steps to reproduce
1. Open codepen. Ensure window height is low enough to cause scrolling of the displayed inputs.
2. Click 'Open Dialog' button
3. Click within one of the inputs to open an inner dialog
4. Click away from the inner dialog. Note the double-scrollbar.

### Expected Behavior
The outer (html/body) scrollbar should not be displayed. Only the fullscreen dialog's scrollbar should be displayed.

### Actual Behavior
The outer (html/body) scrollbar is displayed when the inner dialog is closed.

### Reproduction Link
https://codepen.io/185driver/pen/QZrMPM

### Additional Comments:
1. Ironically, the double-scrollbar returns to a single scrollbar while the inner dialog is being displayed.
2. This is similar to, but not exactly the same as, the recently fixed #2597 bug.

**Comment by johnleider** [comment]
In order to solve this we are going to have to move the scrollbar visibility to the Vuetify object.

**Comment by tomowang** [comment]
I use [vue2-perfect-scrollbar](https://github.com/mercs600/vue2-perfect-scrollbar) to solve this issue (wrap outer html/body with perfect-scollbar).

**Comment by johnleider** [comment]
This is something I still plan to implement and began to here https://github.com/vuetifyjs/vuetify/tree/feat/stack-service . However, it did not make the cut for v2.0

**Comment by keisaac** [comment]
Any news about this?

**Comment by ajlozier** [comment]
I encountered this same issue when I had a dialog opened by an item in a temporary drawer. I wanted to close the drawer when the dialog was opened, but this caused the `.overflow-y-hidden` class to be removed from the html tag, resulting in a double scrollbar. This is how I worked around it:

```
this.drawer = false // drawer model
Vue.nextTick().then(function() {
  document.documentElement.classList.add('overflow-y-hidden')
})
this.dialog = true // dialog model
```

I am sure a similar pattern could be used in other cases like this.

**Comment by mauricio-cantu** [comment]
> I encountered this same issue when I had a dialog opened by an item in a temporary drawer. I wanted to close the drawer when the dialog was opened, but this caused the `.overflow-y-hidden` class to be removed from the html tag, resulting in a double scrollbar. This is how I worked around it:
> 
> ```
> this.drawer = false // drawer model
> Vue.nextTick().then(function() {
>   document.documentElement.classList.add('overflow-y-hidden')
> })
> this.dialog = true // dialog model
> ```
> 
> I am sure a similar pattern could be used in other cases like this.

Worked for me. Thanks!

**Comment by KaelWD** [comment]
Resolved in #14030 by 00e5391a9f9017c21ce3421d70241ff380038a04
